tests/rundb/test_sqldb.py

```diff
@@ -154,61 +159,27 @@ def test_list_tags(db: SQLDB, db_session: Session):
 
 
 def test_projects(db: SQLDB, db_session: Session):
```

Comment:
```suggestion
def test_projects_crud(db: SQLDB, db_session: Session):
```
or "actions", for consistency with tests in other places where you named it so